bigquery/transfer/cloud-client/requirements-test.txt

```diff
@@ -0,0 +1,2 @@
+pytest==5.3.2; python_version > "3.0"
```

Comment:
Do we need both pytest versions for samples we assume will only be run with Python 3?

Reply:
Probably not - I'll update.